ui/imports/shared/popups/addaccount/stores/AddAccountStore.qml

```diff
@@ -119,6 +124,10 @@ BasePopupStore {
         root.addAccountModule.changeSelectedOrigin(keyUid)
     }
 
+    readonly property var validateEnsAsync: Backpressure.debounce(root, 500, function (name, uuid) {
```

Comment:
let's not add the debounce method in store, stores are meant to be stateless. cause what if at another spot you need to call this function with no timeout..

Reply:
Yes, I agree, since stores should just hide the backend, but in this case, since this is not refactoring we don't have adaptors to add it there, guess it will be part of UI improvements.

> what if at another spot you need to call this function with no timeout..

In such cases, until we add adaptor here, we will add a new method when needed, which will be out of debonce. ;)